File: Day18/CoreData/Day27CoreData/BasicCoreData04/Person.m
//
//  Person.m
//  BasicCoreData
//
//  Created by cuong minh on 3/21/12.
//  Copyright (c) 2012 __MyCompanyName__. All rights reserved.
//

#import "Person.h"
#import "Department.h"


@implementation Person

@dynamic dateOfBirth;
@dynamic firstName;
@dynamic fullName;
@dynamic lastName;
@dynamic salary;
@dynamic memberOf;

- (void)awakeFromFetch
{
    [super awakeFromFetch];
    NSMutableString * tmpString = [[NSMutableString alloc] initWithFormat: @"%@ %@", self.firstName, self.lastName];
    self.fullName = [[NSString alloc] initWithString: tmpString];
}
@end

File: Day19/CoreDataSwift/CoreDataSwift/Person.swift
//
//  Person.swift
//  CoreDataSwift
//
//  Created by cuong minh on 12/3/14.
//  Copyright (c) 2014 Techmaster. All rights reserved.
//

import Foundation
import CoreData

class Person: NSManagedObject {

    @NSManaged var email: String
    @NSManaged var firstName: String
    @NSManaged var fullName: String
    @NSManaged var lastName: String
    @NSManaged var cats: NSSet

}
